remove_restore.py

```diff
@@ -56,19 +66,18 @@ def main():
     parser.add_argument('--diff_threshold', default=0.0, help='value above which differences will be added to the base grid')
     parser.add_argument('--plot', action='store_true', help='plot final output before saving')
     parser.add_argument('--output', required=True, help='filename of final output')
+    parser.add_argument('--region_of_interest', required=False, nargs=4, type=float,
+                        help='output region in order <xmin> <xmax> <ymin> <ymax>')
```

Comment:
Would this small clarification be helpful? (Had to jump to various other places in the code to understand what happens when this is not specified)
```suggestion
                        help='(optional) output region in order <xmin> <xmax> <ymin> <ymax>. Defaults to the extent of the base grid.')
```

Reply:
argparse automatically informs the user that a flag is optional (as in this case) but describing the defaults is a good shout.